std/algorithm/searching.d

```diff
@@ -848,7 +848,8 @@ if (isForwardRange!R
     }
 
     //Because of @@@8804@@@: Avoids both "unreachable code" or "no return statement"
-    static if (isInfinite!R) assert(0);
+    static if (isInfinite!R) assert(false, R.stringof ~ "must not be an"
```

Comment:
static if (isInfinite!R) assert(false, R.stringof ~ " must not be an"
Lost a space. ^_^